enhancements/authentication/automated-service-ca-rotation.md

```diff
@@ -102,29 +102,68 @@ and the functionality could be separately extracted for operator reuse.
     original expiry of the pre-rotation CA to refresh without breaking trust.
 
 - The duration of the service CA should be extended from its current value of
-  12 months to a value that exceeds the maximum expected upgrade interval
-  (currently 12 months). A value of 14 months should be sufficient.
-  - Timelines:
-    - With 12 month CA duration:
+  12 months to 26 months and the minimum CA duration should be extended to 13
+  months. These values ensure that pods will be guaranteed to be restarted in
+  a cluster supported by Red Hat before the expiry of the pre-rotation CA.
+  - The timing of upgrades is key to determining CA duration:
+    - Services that do not refresh key material automatically must be
+      restarted after CA rotation.
+    - Upgrades restart all pods, so ensuring that an upgrade takes place
+      after rotation and prior to expiry of the pre-rotation CA guarantees
+      that services will always be using valid key material.
+  - Computing CA duration:
+    - The maximum interval, *I*, between upgrades of a cluster supported by
+      Red Hat is 12 months.
```

Comment:
there's been whispers of an LTS version, wouldn't that break this assumption?

Reply:
afaik there will be no such thing as an LTS release that doesn't still require an upgrade at least once a year if only to a patch release. Any upgrade will do, not just between minor versions. 

@smarterclayton Can you confirm?

Reply:
Added mention of LTS and upgrade to patch release in fixup #2

Reply:
We don't require that customers, "upgrade", but in general customers should be updating production clusters more than once a year; given that we are providing them (at least with micro-updates) weekly updates.